app/assets/stylesheets/_tables.scss

```diff
@@ -0,0 +1,36 @@
+[aria-sort] > * {
+  position: relative;
```

Comment:
This together with the `.visually-hidden` span is causing a weird effect in Firefox on focus:
<img width="958" alt="table-sorting-focus" src="https://cloud.githubusercontent.com/assets/108893/21691009/af9e139c-d36e-11e6-9945-80ac3117012a.png">

I tried to play around to find some solution but nothing I tried was ideal. I don't think it's a big issue. So, maybe just spend 10 minutes with it and ignore it if you cannot find a solution within that time.